src/core/components/questionnaires/cards/CardTemplate.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div v-if="card.template === TRANSITION">
+    <card-transition :title="card.title" />
+  </div>
```

Comment:
pour afficher le bon composant tu peux utiliser `<component` avec la props `:is` dans laquelle tu passes un méthode qui selon la template renvoie le bon composant comme ce qui est fait dans `CardEdition` par exemple

Reply:
Trop styleeeee, je savais pas qu'on pouvait faire ca !